release/release.go

```diff
@@ -151,6 +151,29 @@ func CheckUpstreamRelease(ctx context.Context, client *github.Client, org, repo
 	return releases, nil
 }
 
+func KubernetesGoVersion(ctx context.Context, client *github.Client, version string) (string, error) {
+	var githubError *github.ErrorResponse
+
+	file, _, _, err := client.Repositories.GetContents(ctx, "kubernetes", "kubernetes", ".go-version", &github.RepositoryContentGetOptions{
+		Ref: version,
+	})
+	if err != nil {
+		if errors.As(err, &githubError) {
+			if githubError.Response.StatusCode == http.StatusNotFound {
+				return "", errors.New("failed to find .go-version file in given Kubernetes version")
+			}
+		}
+		return "", err
+	}
+
+	goVersion, err := file.GetContent()
+	if err != nil {
+		return "", errors.New("failed to decode content from .go-version file")
+	}
+
```

Comment:
Since we don't have control of the contents of the file, there might be additional new line characters. Please strip them before returning the string.

Reply:
Done, thank you Brian !